staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -269,20 +271,6 @@ func clearSecurityContext(p *corev1.Pod, containerName string) {
 	})
 }
 
-// setPrivileged configures the containers as privileged.
-func setPrivileged(p *corev1.Pod, containerName string) {
```

Comment:
@verb 
Yes. In fact I initially left `setPrivileged` at first, but it broke lint test as it was not used anywhere.
As you say, it may be used in #119200, but I realize that this is not certain at this point.
So I remove `setPrivileged` at this PR.
I think that it can be revived again if it is necessary for #119200.